integration-tests/contracts/contract_vrf_models.go

```diff
@@ -120,3 +120,7 @@ type VRFBeaconConsumer interface {
 	GetRequestIdsBy(ctx context.Context, nextBeaconOutputHeight *big.Int, confDelay *big.Int) (*big.Int, error)
 	GetRandomnessByRequestId(ctx context.Context, requestID *big.Int, numWordIndex *big.Int) (*big.Int, error)
 }
+
+type BatchBlockhashStore interface {
```

Comment:
I see it's only implemented once by`EthereumBatchBlockhashStore`. Can we skip having this as interface and directly return `EthereumBatchBlockhashStore` in DeployBatchBlockhashStore?

Reply:
I'd prefer we follow the established pattern

Reply:
discussed offline